tests/projects/test_strike_wiring.py

```
@@ -131,3 +131,27 @@ async def test_unquarantine_route_lead_success_and_noauth_failure(app):
             assert body["status"] == "open"
 
             assert await strikes.count_strikes(task_id) == 0
+
+
+@pytest.mark.asyncio
+async def test_release_parks_after_threshold_strikes(app):
+    """Releasing a claimed task records a strike; after STRIKE_THRESHOLD
+    cumulative releases the task is permanently parked."""
+    async with app.router.lifespan_context(app):
+        async with _auth_client(app) as c:
+            project_id, task_id = await _make_project_and_task(c, "strike-park")
```

Ruff (0.16.3)

[warning] Unpacked variable `project_id` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)